assets/src/edit-story/components/panels/animation/directionRadioInput.js

```diff
@@ -0,0 +1,208 @@
+      }
+    }}
+  );
+`;
+
+const Direction = ({ className, direction }) => (
+  <Icon className={className} direction={direction}>
+    <Svg viewBox="0 0 10 11">
+      <path d="M5 11L5 1M5 1L9 5M5 1L1 5" />
+    </Svg>
+  </Icon>
+);
+
+Direction.propTypes = {
+  className: PropTypes.string,
+  direction: PropTypes.oneOf(Object.values(DIRECTION)),
+};
+
+// Must be a styled component to add component selectors in css
+const DirectionIndicator = styled(Direction)``;
+
+const Fieldset = styled.fieldset`
+  position: relative;
+  height: 63px;
+  width: 63px;
+  background-color: ${({ theme }) => theme.colors.bg.workspace};
+  border: 1px solid ${({ theme }) => theme.colors.fg.v9};
+  border-radius: 4px;
+`;
+
+const Figure = styled.div`
+  position: absolute;
+  display: block;
+  top: 50%;
+  left: 50%;
+  width: 18px;
+  height: 26px;
+  background: ${({ theme }) => theme.colors.fg.v9};
+  border-radius: 2px;
+  transform: translate(-50%, -50%);
+`;
+
+const Label = styled.label`
+  position: absolute;
+  cursor: pointer;
+
+  ${({ direction }) => {
+    switch (direction) {
+      case DIRECTION.RIGHT_TO_LEFT:
+        return css`
+          top: 50%;
+          right: 0;
+          transform: translateY(-50%);
+        `;
+      case DIRECTION.TOP_TO_BOTTOM:
+        return css`
+          top: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        `;
+      case DIRECTION.LEFT_TO_RIGHT:
+        return css`
+          top: 50%;
+          left: 0;
+          transform: translateY(-50%);
+        `;
+      default:
+        return css`
+          bottom: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        `;
+    }
+  }}
+
+  ${DirectionIndicator} {
+    stroke: ${({ theme }) => theme.colors.fg.v9};
+    stroke-width: 1px;
+  }
+
+  input:checked ~ ${DirectionIndicator} {
+    stroke: ${({ theme }) => theme.colors.activeDirection};
+    stroke-width: 2px;
+  }
+
+  input:focus ~ ${DirectionIndicator} {
+    outline: 2px auto ${({ theme }) => theme.colors.accent.primary};
+  }
+`;
+
+const hidden = css`
+  position: absolute;
+  opacity: 0;
+  clip-path: polygon(0 0);
+  cursor: pointer;
+`;
+const HiddenLegend = styled.legend`
+  ${hidden}
+`;
+const HiddenInput = styled.input`
+  ${hidden}
+`;
+
+const RadioGroup = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+`;
+
+const camelToPascal = (string) =>
+  string.charAt(0).toUpperCase() + string.slice(1);
+
+const pascalToSentance = (string) => string.replace(/([a-z])([A-Z])/g, '$1 $2');
+
+export const DirectionRadioInput = ({
+  directions = [],
+  onChange,
+  defaultChecked,
+}) => {
+  return (
+    <Fieldset>
+      <Figure />
+      <HiddenLegend>{__('Which Direction?', 'web-stories')}</HiddenLegend>
+      <RadioGroup>
+        {directions.map((direction) => (
+          <Label
+            key={direction}
+            aria-label={sprintf(
+              /* translators: %s: which direction. */
```

Comment:
Maybe give an example? e.g. 


```suggestion
              /* translators: %s: Direction, for example 'top' or 'left'. */
```

Reply:
Actually, it looks like `direction` will bee something like `rightToLeft`, and then you are transforming this to `right-to-left`? But `right-to-left` cannot really be translated? 🤔 

I would change this to:

`__( 'Direction: %s', 'web-stories' )` and then you'll need to make all the directions translatable, e.g. `__('right-to-left')`, etc. And no, `__(direction)` doesn't work :-)

Hope that makes sense.